src/form/types.ts

```diff
@@ -39,82 +40,103 @@ export interface FormLabelProps
     "data-testid"?: string | undefined;
 }
 
+export type FormElementLayoutType = "flex" | "grid";
+
 export interface BaseFormElementProps {
     label?: FormLabelProps | string | undefined;
     errorMessage?: string | undefined;
     "data-error-testid"?: string | undefined;
+    /** The layout type. Values: "flex" | "grid" */
+    layoutType?: FormElementLayoutType | undefined;
 }
 
-export interface FormWrapperProps extends BaseFormElementProps {
+export interface FormWrapperProps extends BaseFormElementProps, ColProps {
```

Comment:
what was the consideration for adding `ColProps` to each interface instead of `BaseFormElementProps`?

Reply:
originally i thought it cannot. Now that i relook, it probably could.